### PR TITLE
feat: add mintPrice to ERC721

### PR DESCRIPTION
**Pull request by mystdeim**

### CODE REVIEW

File: nft/listener/src/main/kotlin/com/rarible/protocol/nft/listener/service/descriptors/erc721/TransferLogsPostProcessor.kt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+        // tx hash -> value of transaction
+        val valueByHash: Map<String, BigInteger> = block.logs.map { it.ethTransaction }
+            .filter { it.value() > BigInteger.ZERO }
+            .associateBy({ tx -> tx.hash().toString() }, { tx -> tx.value() })
+
+        // tx hash -> mintPrice
+        val priceByHash: Map<String, BigInteger?> = ethereumLogs
+            .filter { it.transactionHash in valueByHash.keys }
+            .filter { (it.data as ItemTransfer).isMintTransfer() }
```
Comment (yakischik) [inline nft/listener/src/main/kotlin/com/rarible/protocol/nft/listener/service/descriptors/erc721/TransferLogsPostProcessor.kt:28]: There could be only Transfers? No filtering like is Transfer required?

File: nft/listener/src/main/kotlin/com/rarible/protocol/nft/listener/service/descriptors/erc721/TransferLogsPostProcessor.kt [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            .groupingBy { it.transactionHash }
+            .eachCount()
+
+            // tx hash -> mint count
+            .entries.associate { (hash, count) ->
+                val value = valueByHash[hash]
+                hash to value?.divide(count.toBigInteger())
```
Comment (yakischik) [inline nft/listener/src/main/kotlin/com/rarible/protocol/nft/listener/service/descriptors/erc721/TransferLogsPostProcessor.kt:35]: BigInteger/BigInteger - is it fine, no rounding problems expected?

File: nft/listener/src/test/kotlin/com/rarible/protocol/nft/listener/service/descriptors/erc721/TransferLogsPostProcessorTest.kt [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import scalether.domain.response.Transaction
+import java.math.BigInteger
+import scalether.domain.Address
+
+class TransferLogsPostProcessorTest {
+
+    private val postProcessor = TransferLogsPostProcessor()
+
+    @Test
+    fun `should calculate mintPrice`() {
```
Comment (yakischik) [inline nft/listener/src/test/kotlin/com/rarible/protocol/nft/listener/service/descriptors/erc721/TransferLogsPostProcessorTest.kt:27]: Add test for edge cases (like 0/0)

**Review by yakischik — APPROVED** [review]

File: nft/listener/src/main/kotlin/com/rarible/protocol/nft/listener/service/descriptors/erc721/TransferLogsPostProcessor.kt (hunk 1 above)
Comment (mystdeim) [reply nft/listener/src/main/kotlin/com/rarible/protocol/nft/listener/service/descriptors/erc721/TransferLogsPostProcessor.kt:28]: Yes, because it consumes only ItemTransfer after ERC721TransferLogDescriptor

**Review by yakischik — APPROVED** [review]
